lib/wovnrb.rb

```diff
@@ -51,7 +51,7 @@ def call(env)
       status, res_headers, body = @app.call(headers.request_out)
 
       # disabled by next Rack middleware
-      return output(headers, status, res_headers, body) unless /html/.match?(res_headers['Content-Type'])
+      return output(headers, status, res_headers, body) unless res_headers['Content-Type'].include?('html')
```

Comment:
`.include?` is a supported function. But the previous code was null safe. `res_headers['Content-Type']` can be null

Reply:
How about adding safe navigation, i.e. `res_headers['Content-Type']&.include?('html')`?

Reply:
In any case, it also needs to be done in #229 since I cherry-picked the Rubocop commit to that PR.

Reply:
safe nav would be OK, just for the cases where it can be null

Reply:
I'll do it on #229 then, since we can pull it back when once it gets merged to master, and we the latest master here.